src/platform/guimac.mm

```diff
@@ -371,6 +371,7 @@ @implementation SSView
 
 - (id)initWithFrame:(NSRect)frameRect {
     NSOpenGLPixelFormatAttribute attrs[] = {
+        NSOpenGLPFADoubleBuffer,
```

Comment:
Again a source file change in a CI configuration commit...

Reply:
I was experimenting with this in between builds and I guess I `git add .`'d it

I will clone the repository again for my own experiments and make a separate commit or PR for the change in `guimac.mm`

Reply:
Did you check these as well?
https://github.com/solvespace/solvespace/pull/751#pullrequestreview-512386393